app/mailers/assignment_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class AssignmentMailer < ApplicationMailer
+    
+    def new_assignment
+        @assignment = params[:assignment]
+        @url = ENV['MAIL_HOST']
```

Comment:
I think we need to differentiate between development MAIL_HOST (localhost:3000) and production (the heroku link). Should we create two env variables.

Reply:
ooh all right, would both the env variables refer to localhost:3000 and just be under different names then?

Reply:
Mm I was thinking we would want a dev_mail_host that is local host 3000 and a prod_mail_host car that is https://unloop-staging.herokuapp.com/.

Reply:
got it, I just included a prod_mail_host to link to the heroku, do you have any suggestions on which one to use for the url in the assignment mailer? or do we have to use both

Reply:
Yeah you can do a check 

```Rails.env.production?```